docs/gluon/index.md

```diff
@@ -0,0 +1,72 @@
+# Gluon
+
+The new Gluon interface is available as a library in Apache MXNet and allows developers of all skill levels to prototype, build, and train deep learning models. This interface greatly simplifies the process of creating deep learning models without sacrificing training speed. You can easily get started with the Gluon interface using [this extensive set of tutorials, documentation, and example](https://virtualenv.pypa.io/en/stable/userguide/).
```

Comment:
this should be link to straight-dope? why is it virtualenv guide?

Reply:
Needs to be changed to http://gluon.mxnet.io